6-compiler.js

```diff
@@ -11,10 +11,10 @@ const text = document.getElementById('text');
 button.addEventListener('click', function() {
   let inputValue = input.value.trim();
   
-  let startStr = inputValue.slice( 0,1 ) ;
-  let endStr = inputValue.slice( -1 ) ;
+  // '('で始まり、')'で終わるか確認
+  const inputValueRegex = /\(.*\)/;
```

Comment:
I suggested `/\(.*\)/` before but this only tests if the string contains `(` followed by `)`. They aren't necessarily at the start/end of the string. See if you can modify the regex so that it does test for strings starting with `(` and ends with `)`.

Reply:
Regular expressions are difficult...😣